draft-ietf-lake-edhoc.md

```diff
@@ -579,25 +579,40 @@ In order to reduce round trips and the number of messages, or to simplify proces
 
 EDHOC allows processing of external authorization data (EAD) to be defined in a separate specification, and sent in dedicated fields of the four EDHOC messages (EAD_1, EAD_2, EAD_3, EAD_4). EAD is opaque data to EDHOC.
 
-Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ead_value) as defined below:
+Each EAD field is a CBOR sequence (see {{CBOR}}) consisting of one or more EAD items (ead_label, ? ead_value) as defined below:
 
 ~~~~~~~~~~~ CDDL
-ead = 1* (
+EAD_x = 1* ead
+
+ead = (
   ead_label : int,
-  ead_value : bstr,
+  ? ead_value : bstr,
 )
```

Comment:
? ead_value : bstr,

Do we really want this? This is parsable but  complicates parsing. That you can not pad with 1 byte is not really an issue.

Reply:
Yes.  Indeed, it also provides a way to include an inexpensive "boolean" EAD, i.e., an EAD the mere presence of which already indicates useful information.

Reply:
There is no objection so we do this.